server/series/api.py

```diff
@@ -265,6 +279,105 @@ def revoke_series_invitation(
     return 200, invitation
 
 
+@router.get(
+    "/accept-invitation",
+    response={200: SeriesRegistrationSchema, 400: Response},
+    auth=None,
+)
+def accept_series_invitation_via_mail(
+    request,
+) -> tuple[int, SeriesRegistration | message_response]:
+
+    token = request.GET.get("token")
```

Comment:
nit: You could add this token as a query param,
https://django-ninja.dev/guides/input/query-params/